tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -153,6 +155,34 @@ def test_brand_string(test_microvm_with_api, network_config):
     assert guest_brand_string == expected_guest_brand_string
 
 
+# Some MSR values should not be checked since they can change at Guest runtime.
+# Current exceptions:
+#   * FS and GS change on task switch and arch_prctl.
+#   * TSC is different for each Guest.
+#   * MSR_{C, L}STAR used for SYSCALL/SYSRET; can be different between guests.
+#   * MSR_IA32_SYSENTER_E{SP, IP} used for SYSENTER/SYSEXIT; same as above.
+#   * MSR_KVM_{WALL, SYSTEM}_CLOCK addresses for struct pvclock_* can be different.
+#
+# More detailed information about MSRs can be found in the Intel® 64 and IA-32
+# Architectures Software Developer’s Manual - Volume 4: Model-Specific Registers
+# Check `arch_gen/src/x86/msr_idex.rs` and `msr-index.h` in upstream Linux
+# for symbolic definitions.
+# fmt: off
+MSR_EXCEPTION_LIST = [
```

Comment:
Nit: Can we move this to the beginning of the file for improved readability?

Reply:
I was thinking about this, but decided to leave it just before the first place where it is used. I can imagine that this file expands with new test functions that don't use the list, and the list definition will lack relevance to the neighbouring code.
Do we have a guideline that advises placing data at the top of the file?

Reply:
That makes sense, we can leave this is as. Not a guidance, more like a personal preference.